examples/react-app/frontend/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "frontend",
+  "name": "@architect-examples/react-app",
```

Comment:
I don't think this is a valid name for a docker image. Remove @.

Reply:
That's not the architect.json, it's the package.json. NPM uses `@` for namespacing (see our CLI package.json)